Core/scripts/dirac-install-extension.py

```diff
@@ -0,0 +1,180 @@
+        "The extension can come from another project than the one installed.")
+  print("\nUsage:\n\n  %s <opts> <cfgFile>" % os.path.basename(sys.argv[0]))
+  print("\nOptions:")
+  for cmdOpt in cmdOpts:
+    print("  %s %s : %s" % (cmdOpt[0].ljust(3), cmdOpt[1].ljust(20), cmdOpt[2]))
+
+  sys.exit(0)
+
+
+def loadConfiguration():
+  """
+  It loads the configuration file
+  """
+  optList, args = getopt.getopt(sys.argv[1:],
+                                "".join([opt[0] for opt in cmdOpts]),
+                                [opt[1] for opt in cmdOpts])
+
+  # First check if the name is defined
+  for opt, value in optList:
+    if opt in ('-h', '--help'):
+      usage()
+    elif opt in ("-M", "--defaultsURL"):
+      diracInstall.cliParams.globalDefaults = value
+
+  rConfig = diracInstall.ReleaseConfig(
+      instName=diracInstall.cliParams.installation,
+      globalDefaultsURL=diracInstall.cliParams.globalDefaults)
+  if diracInstall.cliParams.debug:
+    rConfig.debugCB = diracInstall.logDEBUG
+
+  res = rConfig.loadInstallationDefaults()
+  if not res['OK']:
+    diracInstall.logERROR("Could not load defaults: %s" % res['Message'])
+
+  rConfig.loadInstallationLocalDefaults(args)
+
+  for opName in ('release', 'globalDefaults', 'extensions'):
+    try:
+      opVal = rConfig.getInstallationConfig(
+          "LocalInstallation/%s" % (opName[0].upper() + opName[1:]))
+    except KeyError:
+      continue
+
+    if isinstance(getattr(diracInstall.cliParams, opName), six.string_types):
+      setattr(diracInstall.cliParams, opName, opVal)
+    elif isinstance(getattr(diracInstall.cliParams, opName), bool):
+      setattr(diracInstall.cliParams, opName, opVal.lower() in ("y", "yes", "true", "1"))
+    elif isinstance(getattr(diracInstall.cliParams, opName), list):
+      setattr(diracInstall.cliParams, opName, [opV.strip() for opV in opVal.split(",") if opV])
+
+  # Now parse the ops
+  for opt, value in optList:
+    if opt in ('-r', '--release'):
+      diracInstall.cliParams.release = value
+    elif opt in ('-l', '--project'):
+      diracInstall.cliParams.project = value
+    elif opt in ('-e', '--extensions'):
+      for pkg in [p.strip() for p in value.split(",") if p.strip()]:
+        if pkg not in diracInstall.cliParams.extensions:
+          diracInstall.cliParams.extensions.append(pkg)
+
+  if not diracInstall.cliParams.release and not diracInstall.cliParams.modules:
+    diracInstall.logERROR("Missing release to install")
+    usage()
+
+  diracInstall.cliParams.basePath = diracInstall.cliParams.targetPath
+  if diracInstall.cliParams.useVersionsDir:
+    # install under <installPath>/versions/<version>_<timestamp>
+    diracInstall.cliParams.targetPath = os.path.join(
+        diracInstall.cliParams.targetPath, 'versions', '%s_%s' % (diracInstall.cliParams.release, int(time.time())))
+    try:
+      os.makedirs(diracInstall.cliParams.targetPath)
+    except BaseException:
```

Comment:
Shouldn't it be here `OSError` and then some handling and not just `pass`?

Reply:
This is a (reduced) copy from dirac-install.py. Did not want to touch something that works already. But actually this can be further simplified